soc/espressif/esp32s3/Kconfig.amp

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2023 Espressif Systems (Shanghai) Co., Ltd.
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_ESP32S3
+
+menu "AMP config"
+
+config ESP32S3_APPCPU_IRAM_SIZE
+	hex "ESP32S3 APPCPU IRAM size"
+	depends on SOC_ESP32S3_PROCPU || SOC_ESP32S3_APPCPU
+	default 0x10000
```

Comment:
why it was decreased to 64KB?

Reply:
It is the same setup as before due to a different approach. Now both IRAM and DRAM adds to allocate the space.